src/porepy/models/solution_strategy.py

```diff
@@ -228,6 +228,30 @@ def set_discretization_parameters(self) -> None:
 
         """
 
+    def set_solver_statistics(self) -> None:
+        """Set the solver statistics object.
+
+        This method is called at initialization. It is intended to be used to
+        set the solver statistics object(s). Currently, the solver statistics
+        object is related to nonlinearity only. Statistics on linear solvers
```

Comment:
```suggestion
        object is related to nonlinearity only. Statistics on other parts of the solution process, such as linear solvers,
```
No way that fits with the line length requirement, but then the whole paragraph probably would do with a reformatting.